### PR TITLE
README.md: Add Devbox

### PR DESCRIPTION
**Pull request by gmeligio**
It's helpful to work in different shell environments for every project. Similar to Python's virtualenv. It's a layer on top of Nix to work with isolated shells. Quoting the project description: 

> Devbox is a command-line tool that lets you easily create isolated shells and containers. You start by defining the list of packages required by your development environment, and Devbox uses that definition to create an isolated environment just for your application.
> 
> In practice, Devbox works similar to a package manager like yarn – except the packages it manages are at the operating-system level (the sort of thing you would normally install with brew or apt-get).

### CODE REVIEW

**Comment by Mic92** [comment]
Looks good to me. The project has been around for a while and seems popular.

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@
 * [Arion](https://github.com/hercules-ci/arion) - Run `docker-compose` with help from Nix/NixOS.
 * [cached-nix-shell](https://github.com/xzfc/cached-nix-shell) - A `nix-shell` replacement that uses caching to open subsequent shells quickly.
 * [Cachix](https://cachix.org/) - Hosted binary cache service; free for open-source projects.
+* [Devbox](https://github.com/jetpack-io/devbox) - Instant, easy, and predictable development environments
```
Comment (cyntheticfox) [inline README.md:123]: Idk about "instant", but I might recommend emphasizing "portable"  given `devcontainer.json` support.

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Needs punctuation (see CI failure), and consider my line comment, but otherwise looks good.

(Using the curl-piped-to-sh pattern caused some psychic damage to see, but that shouldn't disqualify this PR in any way)

File: README.md (hunk 1 above)
Comment (gmeligio) [reply README.md]: Thank you for reviewing. Added "portable". The "instant" should be related to the caching. That the next time the env starts up is faster in comparison with other package managers (used in the analogy of the project description).

**Pull request merged**